File: pubsub/src/main/java/glitch/pubsub/events/TimeoutUserEvent.java
package glitch.pubsub.events;

import glitch.api.ws.events.IEvent;
import glitch.pubsub.GlitchPubSub;
import glitch.pubsub.Topic;
import glitch.pubsub.object.json.Timeout;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class TimeoutUserEvent extends AbstractMessageEvent<Timeout> implements IEvent<GlitchPubSub> {
    public TimeoutUserEvent(GlitchPubSub client, Topic topic, Timeout data) {
        super(client, topic, data);
    }
}

File: pubsub/src/main/kotlin/glitch/pubsub/events/TimeoutUserEvent.kt
package glitch.pubsub.events

import glitch.api.ws.events.IEvent
import glitch.pubsub.GlitchPubSub
import glitch.pubsub.Topic
import glitch.pubsub.`object`.json.Timeout

class TimeoutUserEvent(client: GlitchPubSub, topic: Topic, data: Timeout) : AbstractMessageEvent<Timeout>(client, topic, data), IEvent<GlitchPubSub> {

}
